subprojects/docs/src/docs/userguide/console.adoc

```diff
@@ -0,0 +1,145 @@
+
+----
+> Task :compileJava
+Note: Some input files use unchecked or unsafe operations.
+Note: Recompile with -Xlint:unchecked for details.
+
+> Task :generateCode
+Generating JAXB classed from XSD files.
+
+> Task :compileJava
+Note: Some input files use or override a deprecated API.
+Note: Recompile with -Xlint:deprecation for details.
+----
+
+[[sec:console_build_progress_bar]]
+==== Build progress bar
+
+The build progress bar gives you a very fast way of knowing if the build will be finished soon. As the build performs work, the progress bar will fill from left to right. At any given time, the build progress bar also renders the current phase of the build lifecycle (see <<sec:build_phases>>) and the overall time spent during the build.
+
+The following examples shows the progress bar during the initialization, configuration and execution phase of the build lifecycle:
+
+----
+<-------------> 0% INITIALIZING [2s]
+<==-----------> 25% CONFIGURING [4s]
+<=========----> 64% EXECUTING [17s]
+----
+
+[[sec:console_work_in_progress_display]]
+==== Work in-progress display
+
+Gradle provides a fined-grained view of the actual work being performed directly underneath the <<sec:console_build_progress_bar,The build progress bar>>. Each line represents a thread or process that can perform work in parallel--resolving dependencies, executing a task and running tests. If an available worker is not being used then it is marked with `IDLE`. The number of available workers defaults to the number of processors on the machine executing the build. You can adjust the number of workers through the <<gradle_command_line,command line>> option `--max-workers`.
+
+The following portion of the console output shows the work in-progress display with 8 concurrent workers:
+
+----
+<==========---> 77% EXECUTING [10s]
+> :codeQuality:classpathManifest > Resolve dependencies :codeQuality:runtimeClasspath
+> :ivy:classpathManifest > Resolve dependencies :ivy:runtimeClasspath
+> IDLE
+> :antlr:classpathManifest > Resolve dependencies :antlr:runtimeClasspath
+> :scala:compileJava > Resolve dependencies :scala:compileClasspath
+> :buildInit:classpathManifest > Resolve dependencies :buildInit:runtimeClasspath
+> :jacoco:classpathManifest > Resolve dependencies :jacoco:runtimeClasspath
+> IDLE
+----
+
+[[sec:console_build_result]]
+==== Build result
+
+At the end of the build, Gradle will display the result of the build (successful or failed) and the number of tasks that performed work and avoided work. The build result also displays the overall elapsed time it took to execute the build. The number of tasks that performed work provides an indication of how out-of-date or busy the build was.
+
+----
+BUILD SUCCESSFUL in 2m 10s
+411 actionable tasks: 381 executed, 30 up-to-date
```

Comment:
:x: define "actionable" as a task having "actions" (e.g. `:build` does not have actions). Seems like we could link to more info for that.